File: baekjoon/silver/1676/1676.py
two = 0
five = 0
for i in range(2, int(input())+1):
    while i % 2 == 0:
        i //= 2
        two += 1
    while i % 5 == 0:
        i //= 5
        five += 1
print(min(two, five))
    

File: baekjoon/silver/1676/1676.go
package main

import "fmt"

func main() {
	var n int

	fmt.Scan(&n)
	t, f := 0, 0
	for i := 2; i < n+1; i++ {
		v := i
		for v%2 == 0 {
			v /= 2
			t++
		}
		for v%5 == 0 {
			v /= 5
			f++
		}
	}
	fmt.Println(min(t, f))
}

func min(a, b int) int {
	if a < b {
		return a
	}
	return b
}
